src/SDKs/Batch/Management/Management.Batch/Properties/AssemblyInfo.cs

```diff
@@ -8,8 +8,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Batch Management Library")]
 [assembly: AssemblyDescription("Provides management functions for Microsoft Azure Batch services.")]
 
-[assembly: AssemblyVersion("5.0.0.0")]
-[assembly: AssemblyFileVersion("5.0.0.0")]
+[assembly: AssemblyVersion("5.1.0.0")]
```

Comment:
I just did some reading and it sounds like we shouldn't change `AssemblyVersion` here, so I will leave it 5.0.0.0